apps/daimo-bot/vercel.json

```diff
@@ -0,0 +1 @@
+{ "version": 2, "rewrites": [{ "source": "/(.*)", "destination": "/src/api" }] }
```

Comment:
@nalinbhardwaj btw, switching from self-hosted Hubble stream to Neynar webhook, will use Vercel instead of Render